sound/soc/sof/Kconfig

```diff
@@ -41,6 +41,15 @@ config SND_SOC_SOF_OF
 	  required to enable i.MX8 devices.
 	  Say Y if you need this option. If unsure select "N".
 
+config SND_SOC_SOF_COMPRESS
+	bool "SOF enable compress interface"
```

Comment:
tristate?

Not sure why we'd want that code to be built-in?

Reply:
@plbossart no strong opinion on this. I just looked on how compress probe symbol was defined.